config/karma.base.js

```diff
@@ -68,7 +69,7 @@ const config = {
   reporters: ['spec' /*, 'saucelabs' */],
 
   // web server port
-  port: 8080,
+  port: 8089,
```

Comment:
I don't have a huge issue w/ this, but is it necessary to change the ports of **all** of the other tests? Or can your local firestore instance just run on a different port?

Reply:
Originally I just did it for Firestore, but it seemed weird to be inconsistent.  If we use two different ports it means somebody trying to run all of the tests will be twice as likely to have a conflict. :-)  

FWIW, 8080 seems like a problematic default since that's the default port for many commonly used web servers (e.g. the http-server npm module).  So changing to 8089 will hopefully reduce the chances people will run into trouble. And FWIW, the failure mode when there's a conflict is pretty poor; karma just times out trying to attach to ChromeHeadless and tries repeatedly...

If you really prefer, I can move this to just the firestore karma config though.  It makes no difference to me.

Reply:
Eh, alright.